### PR TITLE
Quick PR to add my changes to upstream repo

### PR DESCRIPTION
**Pull request by ChrisJabb21**

### CODE REVIEW

**Comment by ChrisJabb21** [comment]
Added Maven Wrapper and removed unnecessary files for project.

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                url = mavenWrapperProperties.getProperty(PROPERTY_NAME_WRAPPER_URL, url);
+            } catch (IOException e) {
+                System.out.println("- ERROR loading '" + MAVEN_WRAPPER_PROPERTIES_PATH + "'");
+            } finally {
+                try {
+                    if(mavenWrapperPropertyFileInputStream != null) {
+                        mavenWrapperPropertyFileInputStream.close();
+                    }
+                } catch (IOException e) {
+                    // Ignore ...
```
Comment (natashang) [inline .mvn/wrapper/MavenWrapperDownloader.java:72]: +error handling - system.out.println(explanation of exception here)

File: src/main/java/com/habibank/controller/AuthController.java [hunk 2]
```diff
@@ -1,135 +1,135 @@
-package com.habibank.controller;
+// package com.habibank.controller;
```
Comment (natashang) [inline src/main/java/com/habibank/controller/AuthController.java]: all of these lines are commented out - can we delete the file if it is not needed?

File: src/main/web-app/src/router/AppRouter.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (natashang) [inline src/main/web-app/src/router/AppRouter.js:1]: we can keep this for now as an additional resource should we need it

**Review by natashang — CHANGES_REQUESTED** [review]
just a couple feedback notes to tweak. see comments

**Review by natashang — APPROVED** [review]
looks good to me

**Pull request merged**